vmas/interactive_rendering.py

```diff
@@ -117,7 +119,17 @@ def _cycle(self):
                 ] = self.u2[
                     : self.agents[self.current_agent_index2].dynamics.needed_action_size
                 ]
-            obs, rew, done, info = self.env.step(action_list)
+
+            action_tensors = [
+                torch.tensor(
+                    act,
+                    device=self.env.device,
+                    dtype=torch.float32,
+                ).reshape(1, self.env.get_agent_action_size(agent))
+                for act, agent in zip(action_list, self.env.agents)
+            ]
+
+            obs, rew, done, info = self.env.step(action_tensors)
```

Comment:
After the removal of the wrapper you need to add the vmas nested indexing util here for it to run i think